src/test/java/com/bravo/user/dao/model/mapper/ResourceMapperTest.java

```diff
@@ -28,4 +30,11 @@ void convertUserTest(
     Assertions.assertEquals(userReadDto, new ResourceMapper(
         new DefaultMapperFactory.Builder().build().getMapperFacade()).convertUser(user));
   }
+  
+  void converAddressTest(
+	      @ConvertWith(MapperArgConverter.class) Address address,
+	      @ConvertWith(MapperArgConverter.class) AddressReadDto addressReadDto) {
+	    Assertions.assertEquals(addressReadDto, new ResourceMapper(
+	        new DefaultMapperFactory.Builder().build().getMapperFacade()).convertAddress(address));
```

Comment:
Instead of duplicating this logic perhaps it would be better as a variable instead. 

```
private static final ResourceMapper MAPPER = new DefaultMapperFactory.Builder().build().getMapperFacade();
```